### ISSUE TITLE
exporters::loadbalancing: requires a non-empty "endpoint" in version  0.95.0

### ISSUE THREAD

**Issue by firecow** [open-telemetry/opentelemetry-collector-contrib#31388]
## Description
validate reports `exporters::loadbalancing: requires a non-empty "endpoint"` in version 0.95.0
## Steps to Reproduce
Run otel-collector validate on a config file that has a loadbalancing exporter
## Expected Result
I would expect the validation to succeed.
## Actual Result
It throws a validation error

### Collector version

0.95.0

## Environment
Docker image from dockerhub otel/opentelemetry-collector-contrib:0.95.0

### OpenTelemetry Collector configuration

```yaml
---
receivers:
  otlp:
    protocols:
      http:

exporters:
  debug:

  otlphttp:
    endpoint: ${env:ELASTIC_APM_ENDPOINT}
    headers:
      "Authorization": "Bearer ${env:ELASTIC_APM_SECRET_TOKEN}"

  loadbalancing:
    routing_key: traceID
    protocol:
      otlp:
        timeout: 1s
        tls:
          insecure: true
    resolver:
      dns:
        hostname: otel_collector_layer_two

processors:
  batch:

extensions:

service:
  pipelines:
    traces:
      receivers: [otlp]
      processors: []
      exporters: [loadbalancing]
    metrics:
      receivers: [otlp]
      processors: [batch]
      exporters: [otlphttp]
    logs:
      receivers: [otlp]
      processors: [batch]
      exporters: [otlphttp]
```

### Log output

```shell
Reinitialized existing Git repository in /data/gitlab-runner/builds/W9JEWgWC/0/cego/otel-collector/.git/
Checking out 6903c0c8 as detached HEAD (ref is renovate/otel-opentelemetry-collector-contrib-0.x)...
Skipping Git submodules setup
Executing "step_script" stage of the job script
00:08
$ docker run -q -e ELASTIC_APM_ENDPOINT -e ELASTIC_APM_SECRET_TOKEN --rm -v $(pwd)/collector-config-layer-one.yml:/etc/collector-config-layer-one.yml ${OTEL_COLLECTOR_IMAGE_REF} --config=/etc/collector-config-layer-one.yml validate
Error: exporters::loadbalancing: requires a non-empty "endpoint"
2024/02/22 19:07:12 collector server run finished with error: exporters::loadbalancing: requires a non-empty "endpoint"
```

### Additional context

I've read the release notes and looked at the loadbalancing examples, but no where is endpoint mentioned in conjunction with loadbalancing.

**Comment by willthames** [comment]
Looks like this is the same as #31371, fixed (but not yet released) in #31381

**Comment by crobert-1** [comment]
Closing as duplicate of #31371 👍
